dictionary.json

```diff
@@ -3,6 +3,70 @@
   "description": "The Attribute Dictionary defines attributes and includes references to the events and objects in which they are used.",
   "name": "dictionary",
   "attributes": {
+    "activities": {
+      "type": "activities",
+      "caption": "Activities"
+    },
+    "authentication_activities": {
+      "caption": "Authentication Activities",
+      "description": "Authentication Activities events report authentication session activity.",
+      "is_array": true,
+      "type": "authentication_related_activity"
+    },
+    "dns_activities": {
+      "caption": "DNS Activities",
+      "description": "DNS Activities report DNS queries and answers activity.",
+      "is_array": true,
+      "type": "dns_related_activity"
+    },
+    "email_activities": {
+      "caption": "Email Activities",
+      "description": "Email Activity reports mail activities",
+      "is_array": true,
+      "type": "email_related_activity"
+    },
+    "filesystem_activities": {
+      "caption": "File System Activities",
+      "description": "File System Activity events report when a process performs an action on a file or folder.",
+      "is_array": true,
+      "type": "file_related_activity"
+    },
+    "http_activities": {
+      "caption": "HTTP Activities",
+      "description": "HTTP Activities events report HTTP connection and traffic information.",
+      "is_array": true,
+      "type": "http_related_activity"
+    },  
+    "memory_activities": {
+      "caption": "Memory Activities",
+      "description": "Memory Activities report loads/unload of modules or libraries",
```

Comment:
Copy-n-paste error from the `module_activities` attribute. I'm guessing you meant to base this description on the description of the `Memory Activity` event so it should be something like:

> Memory Activities report when a process has memory allocated, read/modified, or other manipulation activities - such as a buffer overflow or turning off data execution protection (DEP).